lib/v2/components/copy_link_row_light.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+
+/// Copy Link
+///
+/// Used to display a row with a Hash and a copy Icon at the end
+
+class ShareLinkRowLight extends StatelessWidget {
```

Comment:
why do we need two of these? a light and a dark?

Reply:
Sorry forgot designs were change, now we only need one. I remove the other one.